src/decorators.ts

```diff
@@ -36,6 +36,15 @@ export function ToBoolean() {
     });
 }
 
+/**
+ * Cast value to integer type
+ */
+export function ToInteger(radix?: number) {
```

Comment:
I think we can provide default radix as 10

Reply:
It is. Just the same as `parseInt` 
![image](https://user-images.githubusercontent.com/14863908/85842947-21a46780-b7a9-11ea-9dbb-f3261d063870.png)

Reply:
I mean to set explicitly default value as 10. Because almost in all cases we use 10 radix

Reply:
Check